src/utils/tracked-controls.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Enumerate gamepads and apply callback function to each.
+ *
+ * @param {object} callback - callback function that takes gamepad as argument.
+ * @param {object} idPrefix - prefix to match in gamepad id, if any.
+ */
+module.exports.enumerateGamepads = function (callback, idPrefix) {
+  var gamepads = navigator.getGamepads && navigator.getGamepads();
```

Comment:
IIRC this is used as an abstraction in src/systems/tracked-controls, so that if we need to make faux gamepads for controllers that don't enumerate through gamepad API, this is the only place that needs to know.